src/axios.ts

```diff
@@ -3,12 +3,19 @@ import http from 'http';
 import https from 'https';
 import emitter from './emitter';
 
+// For a compatibility with Snyk CLI
+export declare interface Global extends NodeJS.Global {
```

Comment:
Question: maybe I am lacking some understanding but why do you have to extend the global object? Is this because typescript does not define the property that is already there?

Reply:
Global is an Interface with defined properties and TypeScript interfaces are closed for additions. So we need to create our own interface.